util-images/request-benchmark/main.go

```diff
@@ -38,19 +38,58 @@ const (
 	HealthCheckRequests = 10
 	NamespaceTmpl       = "%namespace%"
 )
+type ContentType string
+
+const (
+	JSONContentType  ContentType = "json"
+	ProtoContentType ContentType = "proto"
+	CBORContentType  ContentType = "cbor"
+	YAMLContentType  ContentType = "yaml"
+)
 
 var (
-	inflight  = flag.Int("inflight", 1, "Benchmark inflight (number of parallel requests being made to the apiserver")
-	namespace = flag.String("namespace", "", "Replace %namespace% in URI with provided namespace")
-	URI       = flag.String("uri", "", "Request URI")
-	verb      = flag.String("verb", "GET", "A verb to be used in requests.")
-	qps       = flag.Float64("qps", -1, "The qps limit for all requests")
+	inflight    = flag.Int("inflight", 1, "Benchmark inflight (number of parallel requests being made to the apiserver")
+	namespace   = flag.String("namespace", "", "Replace %namespace% in URI with provided namespace")
+	URI         = flag.String("uri", "", "Request URI")
+	verb        = flag.String("verb", "GET", "A verb to be used in requests.")
+	qps         = flag.Float64("qps", -1, "The qps limit for all requests")
+	contentType = ContentType("") 
```

Comment:
What was the default before this change? Json or proto? Can we set is a default?

Reply:
Empty content type defaults to JSON.